terraform/modules/frontdoor/maintenance_page.tf

```diff
@@ -0,0 +1,33 @@
+
+resource "aws_s3_bucket" "maintenance_page_bucket" {
+  bucket        = "${var.environment_name}-maintenance-page-bucket"
+}
+
+resource "aws_s3_bucket_public_access_block" "maintenance_page_bucket_public_access" {
```

Comment:
Actually we probably don't need this, but it's mainly there to question if it's ok...

It's in the blog post with everything set to true.

But the AWS documenation looks like you need to make everything public to use the s3 static website endpoint

https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteAccessPermissionsReqd.html